Makefile

```diff
@@ -76,11 +76,9 @@ run:
 	${DOCKER_COMPOSE} up --abort-on-container-exit --exit-code-from app; ${DOCKER_COMPOSE} down
 
 dev: export TARGET = dev
-dev: ## Build and runs the container for development
-	make --jobs=4 install build stop
+dev: install build stop ## Build and runs the container for development
 	make run
 
 prod: export TARGET = prod
-prod: ## Builds and runs the container for production
-	make --jobs=2 build stop
+prod: build stop ## Builds and runs the container for production
 	make run
```

Comment:
could move things around with intermediate targets to ensure `run` is a prerequisite as well, but seems harmless at the moment being a single command.

Reply:
Does still end up with `make[1]: warning: -jN forced in submake: disabling jobserver mode.`

Reply:
It could be because this is using `make` rather than `$(MAKE)`; but do we want to go with the recursive make route or completely through prerequisites?